src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -480,6 +506,16 @@ namespace Microsoft.Management.Deployment
         String CorrelationData;
         /// A string that will be passed to the source server if using a REST source
         String AdditionalPackageCatalogArguments;
+
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 2)]
+        {
+            // The set of allowed Architectures, in preference order, that will be considered for
+            // the install operation.  The default set is empty, which allows any architecture
+            // to be used.  Unknown can also be used to set a preference rather than a requirement,
+            // such as { X86, Unknown }, which will prefer X86 if it is available and applicable,

```

Comment:
>Unknown [](http://example.com/codeflow?start=30&length=7)

The name "Unknown" seems not accurate to denote "I'm ok with other defaults" and conflicts with the "Architecture::unknown"meaning in winget , looks like we are redefining the behavior of what Architecture::Unknown means

Reply:
There is not another value within `Windows.System.ProcessorArchitecture` that is not also a value in our internal `Architecture` enum that represents a legitimate installer architecture.  `Neutral` could also work, but is an actual installer architecture.

I had originally created my own enum and used the value `Any` in this role, but I don't like creating yet another enum for this when multiple exist.